ResearchKit/Common/ORKPasscodeStepViewController.m

```diff
@@ -39,9 +39,14 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 #import <AudioToolbox/AudioToolbox.h>
 #import <LocalAuthentication/LocalAuthentication.h>
 
+static CGFloat const kForgotPasscodeVerticalPadding     = 50.0f;
```

Comment:
I always prefer to use constraints, I had trouble because they are using auto resizing masks on other views in this VC, and constraints and those don't play well together.